.pre-commit-config.yaml

```diff
@@ -1,4 +1,15 @@
 repos:
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: "v4.6.0"
+    hooks:
+      - id: check-case-conflict
+      - id: check-merge-conflict
+      - id: check-toml
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
```

Comment:
There is [something in docs](https://github.com/pre-commit/pre-commit-hooks/tree/main?tab=readme-ov-file#trailing-whitespace) about a potential problem with markdown. Should we add an option they mentined in docs?

Reply:
Good idea, done!